CHANGES/1494.feature

```diff
@@ -0,0 +1 @@
+Updated our OCI MANIFEST SCHEMA to comply with the changes from OCI Image Manifest Specification.
```

Comment:
```suggestion
Updated  the OCI manifest schema validation to comply with the OCI Image Manifest Specification.
```

Also, can we add a disclaimer about deprecating `ADDITIONAL_OCI_ARTIFACT_TYPES` here? It should be visible because we are going to kill it on the next release.